dags/experiments_live.py

```diff
@@ -52,28 +52,44 @@
 
 
     # list of datasets to execute query for and export
-    experiment_datasets = [
+    experiment_enrollment_datasets = [
         "moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_other_events_overall_v1",
         "moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_cumulative_population_estimate_v1",
         "moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_overall_v1",
         "moz-fx-data-shared-prod.telemetry_derived.experiment_unenrollment_overall_v1",
+    ]
+
+    experiment_enrollment_export = gke_command(
+        task_id="experiment_enrollment_export",
+        command=[
+            "python", "experiments_monitoring_data_export/export.py",
+            "--datasets",
+        ] + experiment_enrollment_datasets,
+        docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/experiments-monitoring-data-export:latest",
+        dag=dag,
+    )
+
+    # search data export
+    # todo: use materialized views
```

Comment:
I'll migrate these queries once the enrollments data monitoring is up and running